pol-core/bscript/executor.cpp

```diff
@@ -3110,7 +3115,30 @@ void Executor::ins_bitwise_not( const Instruction& /*ins*/ )
 void Executor::ins_funcref( const Instruction& ins )
 {
   ValueStack.push_back( BObjectRef(
-      new BObject( new BFunctionRef( ins.token.lval, ins.token.type, scriptname() ) ) ) );
+      new BObject( new BFunctionRef( ins.token.lval, ins.token.type, scriptname(), {} ) ) ) );
+}
+
+void Executor::ins_functor( const Instruction& ins )
+{
+  int parameter_count = static_cast<BLong*>( ValueStack.back()->impptr() )->value();
```

Comment:
passert_always(ValueStack.back()->Isa(Blong));
Would I think not hurt and shows that something is broken instead of complete undefined behaviour. Two lines later I suggest the same